src/screens/enter-password/enter-password.tsx

```diff
@@ -47,7 +50,16 @@ export const EnterPassword = () => {
   const isBiometryAvailable = isDefined(biometryType) && biometricsEnabled;
   const biometryIconName = biometryType === 'FaceID' ? IconNameEnum.FaceId : IconNameEnum.TouchId;
 
-  const onSubmit = ({ password }: EnterPasswordFormValues) => void (!isDisabled && unlock(password));
+  const onSubmit = ({ password }: EnterPasswordFormValues) => {
+    if (!isDisabled) {
+      dispatch(setLoadingAction(true));
+      unlock(password);
+    }
+  };
+
+  useEffect(() => {
+    dispatch(setLoadingAction(false));
+  }, []);
```

Comment:
why do you need this here?

Reply:
at least one point in application to prevent freezed loading